persistence/trees/trees.go

```diff
@@ -219,7 +224,11 @@ func (t *treeStore) getStateHash() string {
 
 	// Convert the array to a slice and return it
 	// REF: https://stackoverflow.com/questions/28886616/convert-array-to-slice-in-go
-	return hex.EncodeToString(stateHash[:])
+	h := stateHash[:]
+
+	t.logger.Info().Msgf("#️⃣ calculated state hash: %s", hex.EncodeToString(h))
```

Comment:
```suggestion
        hexHash := hex.EncodeToString(h)
	t.logger.Info().Msgf("#️⃣ calculated state hash: %s", hexHash)
```
Then same var in return